docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -10,9 +12,100 @@ which conforms to the schema defined below. You may find it easier to
+
+.. warning:: Why do some modules have imports at the bottom of the file
+
+  If you look at some existing older modules, you may find imports at the bottom of the file. Do not copy that idiom into new modules as it is a historical oddity due to how modules used to be combined with libraries. Over time we're moving the imports to be in their proper place.
+
+
+
+ANSIBLE_METADATA Block
+----------------------
+
+ANSIBLE_METADATA contains information about the module for use by other tools. At the moment, it informs other tools about what class of people maintains the module and to what degree users can rely on a module's behaviour remaining the same over time.
+
+For new modules the following block can be simply added into your module::
+
+   ANSIBLE_METADATA = {'status': ['preview'],
+                       'supported_by': 'community',
+                       'version': '1.0'}
+
+.. warning:: version field
+
+   This is the version of the ``ANSIBLE_METADATA`` schema, *not* the version of the module.
+
+.. warning::
+
+   Promoting a module's ``status`` or ``supported_by`` status should only be done by members of the Ansible Core Team.
+
+Version 1.0 of the metadata
++++++++++++++++++++++++++++
+
+Structure
+`````````
+
+Format::
+
+  ANSIBLE_METADATA = {
+      'version': '1.0',
+      'supported_by': 'core|community|core_curated',
+      'status': ['stableinterface|preview|deprecated|removed']
+  }
+
+Fields
+``````
+
+:version: An “X.Y” formatted string. X and Y are integers which
+   define the metadata format version. Modules shipped with Ansible are
+   tied to an Ansible release so we will only ship with a single version
+   of the metadata. We’ll increment Y if we add fields or legal values
+   to an existing field. We’ll increment X if we remove fields or values
+   or change the type or meaning of a field.
+:supported_by: This field records who supports the module.
+   Default value is community. Values are:
+
+   :core: maintained by the Ansible core team. Core team will fix
+      bugs, add new features, and review PRs.
+   :community: This module is maintained by the community at large.
```

Comment:
Suggest "This module is maintained by the community at large, which is responsible for fixing bugs, adding new features, and reviewing changes."